doc/spec/validation.rst

```diff
@@ -0,0 +1,68 @@
+Validation
+======================
+
+Validating causal estimates is inherently challenging, as the true counterfactual outcome for a given treatment is
+unobservable. However, there are several checks and tools available in EconML to help assess the credibility of causal
+estimates.
+
+
+Sensitivity Analysis
+---------------------
+
+For many EconML estimators, unobserved confounding can lead to biased causal estimates.
+Moreover, it is impossible to prove the absence of unobserved confounders.
+This is a fundamental problem for observational causal inference.
+
+To mitigate this problem, EconML provides a suite of sensitivity analysis tools,
+based on `Chernozhukov et al. (2022) <https://www.nber.org/papers/w30302>`_,
```

Comment:
One more comment - consider adding this paper to the references page and linking to that here instead.